test/dhcp_isc/server_isc_test.rb

```diff
@@ -64,15 +64,15 @@ def test_sparc_host_quirks
     assert_equal [], @dhcp.send(:solaris_options_statements, {})
 
     assert_equal [
-      %q{option SUNW.JumpStart-server \"192.168.122.24:/Solaris/jumpstart\";},
-      %q{option SUNW.install-path \"/Solaris/install/Solaris_5.10_sparc_hw0811\";},
-      %q{option SUNW.install-server-hostname \"itgsyddev807.macbank\";},
-      %q{option SUNW.install-server-ip-address 192.168.122.24;},
-      %q{option SUNW.root-path-name \"/Solaris/install/Solaris_5.10_sparc_hw0811/Solaris_10/Tools/Boot\";},
-      %q{option SUNW.root-server-hostname \"itgsyddev807.macbank\";},
-      %q{option SUNW.root-server-ip-address 192.168.122.24;},
-      %q{option SUNW.sysid-config-file-server \"192.168.122.24:/Solaris/jumpstart/sysidcfg/sysidcfg_primary\";},
-      %q{vendor-option-space SUNW;}
+      'option SUNW.JumpStart-server \"192.168.122.24:/Solaris/jumpstart\";',
+      'option SUNW.install-path \"/Solaris/install/Solaris_5.10_sparc_hw0811\";',
+      'option SUNW.install-server-hostname \"itgsyddev807.macbank\";',
+      'option SUNW.install-server-ip-address 192.168.122.24;',
+      'option SUNW.root-path-name \"/Solaris/install/Solaris_5.10_sparc_hw0811/Solaris_10/Tools/Boot\";',
+      'option SUNW.root-server-hostname \"itgsyddev807.macbank\";',
```

Comment:
I think the meaning of the escapes may change here, as `'\"'` is the value `\"` while `%q{\"}` is `"`.